backend/inspirehep/editor/authorlist_utils.py

```diff
@@ -8,9 +8,13 @@
     r"(\\255|\u02D7|\u0335|\u0336|\u2212|\u2013|\u002D|\uFE63|\uFF0D)", re.UNICODE
 )
 re_multiple_space = re.compile(r"\s{2,}", re.UNICODE)
+re_multiple_comma = re.compile(r", ,", re.UNICODE)
 re_potential_key = re.compile(r"^(?:\d|[^\w.'-])+$", re.UNICODE)
-re_trailing_nonword = re.compile(r"((?:\d|[^\w,.'-])+ )", re.UNICODE)
+re_trailing_nonword = re.compile(r"((?:\d|[^\w,.'])+ |\d+(\.\d+)?)", re.UNICODE)
 re_symbols = re.compile(r"[^\w ]", re.UNICODE)
+re_is_orcid = re.compile(r"^(orcid:)?\d{4}-\d{4}-\d{4}-\d{3}[0-9X]$")
+re_is_email = r"\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b"
+re_replaceble_char = re.compile(r"\sand\s|\n|,", re.UNICODE)
```

Comment:
To unify the separation of authors and affiliations.
There might be text containing ```and``` instead of ```,``` and also ```\n``` might be used as separation.
Examples:
```
"F. Durães1, A.V. Giannini2, V.P. Gonçalves3,4 and F.S. Navarra2\n"
```
or
```
"C. Patrignani\n"
"1\n"
 "K. Agashe\n"
 "2\n"
```